scanners/nmap/parser/parser.js

```diff
@@ -21,7 +21,7 @@ function transformToFindings(hosts) {
 
     return openPorts.map(openPort => {
       return {
-        name: openPort.service,
+        name: openPort.service || `Open Port: ${openPort.port}`,
```

Comment:
I would prefer to use the v3 release to make this more consistent and use the `Open Port: ${openPort.port}` format for all open port findings. 

Maybe with the openPort.service appended in brackets if it exists: 
like "Open Port: 8080 (http-apache)" or "Open Port: 10250"

Reply:
I implemented it, have a look at the recent changes